e3db-crypto-interface/src/main/java/com/tozny/e3db/E3DBCryptoException.java

```diff
@@ -0,0 +1,18 @@
+package com.tozny.e3db;
+
+/**
+ * An exception related to underlying cryptography libraries for tasks such as keygen, key-management, encryption or decryption
+ */
+public class E3DBCryptoException extends Exception {
```

Comment:
Why is extending Exception such a tedious process in creating different constructors...?